handbook/ops/bizops/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# BizOps onboarding
+
+Welcome! We're so excited to have you join the BizOps team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Week 1
+
+### Get set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
```

Comment:
Could you move this down to wherever seems like the best place in the list? I think it makes sense to start with the high-level pieces (values, strategy, etc.), and that the OKRs might be a bit confusing without that context)